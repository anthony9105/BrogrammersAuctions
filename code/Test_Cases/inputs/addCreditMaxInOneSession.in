login adminUser1 ^^^^
addcredit
adminUser1
1001
cancel please
logout
close