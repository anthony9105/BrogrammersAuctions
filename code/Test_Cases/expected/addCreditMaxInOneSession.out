Login successful
Enter username of account to add credit to
Enter credit to add to user
Error. No more than $1000.00 can be added in a given session
Enter credit to add to user
Transaction cancelled
logout successful